src/api/cicd/scripts/keycloak/users.sh

```diff
@@ -6,7 +6,12 @@
 # KC_CLIENT_ID="admin-cli"
 # KC_ADMIN_USER="xxxxx@example.com"
 # KC_ADMIN_PASSWORD="xxxxx"
-KC_JWT=$(curl -s -X POST "${KC_BASE_URL}/realms/${KC_ADMIN_REALM}/protocol/openid-connect/token" \
+
+echo "${KC_CLIENT_ID}"
+echo "${KC_ADMIN_USER}"
+echo "${KC_ADMIN_PASSWORD}"
```

Comment:
I don't think we should be printing keycloak admin credentials

Reply:
Definitely not